content/chat/index.adoc

```diff
@@ -0,0 +1,79 @@
+---
+layout: simplepage
+title: Chat
+---
+
+The Jenkins community discusses various project-related topics in multiple channels on the http://www.freenode.net[Freenode] IRC network.
+
+For information on how to connect, see the http://freenode.net/[Freenode] website.
```

Comment:
Maybe move (or copy) the webchat.freenode.net link from the `#jenkins` section here?